### DISCUSSION TITLE
Issue with Redis Vector Store

### DISCUSSION THREAD

**Discussion by ashokrayapati** [run-llama/llama_index#8115] [General]
set_global_service_context(service_context = service_context)
vector_store=RedisVectorStore(index_name=folderpath[i],index_prefix="llama_index",redis_url=configdata["REDIS_URL"],overwrite=False)
loaded_index = VectorStoreIndex.from_vector_store(vector_store,service_context=service_context)
query_engine = loaded_index.as_query_engine(similarity_top_k= 3,streaming=True)    
response = query_engine.query(input_txt)    
 i just went through https://gpt-index.readthedocs.io/en/latest/examples/vector_stores/RedisIndexDemo.html
 but i haven't found any solution for this can you give me a quick idea about this.?

**Comment by logan-markewich** [comment]
What is the issue though?
